rachelnavarro/step/portfolio/src/main/webapp/script.js

```diff
@@ -24,8 +25,28 @@ function drawCardOfTheDay() {
   tarotDeck.innerText = card;
 }
 
-function displayGreeting() {
-  fetch('/data').then(response => response.text()).then((greeting) => {
-    document.getElementById('greeting-container').innerText = greeting;
+/** Fetches and loads all previous comments in an unordered list. */
+function displayComments() {
+  fetch('/data').then(response => response.json()).then((allcomments) => {
+    var i;
+    for (i = 0; i < allcomments.length; i++) {
+      var datetime = getDateTime();
+      createCommentEntry(allcomments[i].name + ' said "' + allcomments[i].body + "' at " + datetime);
```

Comment:
Are the quotes here correct?

Reply:
I would use a format string, https://stackoverflow.com/questions/610406/javascript-equivalent-to-printf-string-format/32202320#32202320

It is easier to read :)

Something like \`${allComments[i].name} said "${allComments[i].body}" ... etc\`

Also rename allcomments to allComments.